cpp/oneapi/dal/algo/kmeans/backend/cpu/infer_kernel.cpp

```diff
@@ -103,10 +103,9 @@ static infer_result<Task> call_daal_kernel(const context_cpu& ctx,
             .compute(input, output, &par);
     }));
 
-    if (desc.get_result_options().test(result_options::compute_assignments)) {
```

Comment:
please, in case that we dont use it, it makes sense to remove this optional result from descriptor

Reply:
done